.github/workflows/tests.yml

```diff
@@ -206,6 +209,32 @@ jobs:
         stdbuf -o 0 -e 0 prove -j$JOBS t/openssl/ssl/ 2>&1 | grep -v "Connection refused" | grep -v "Retry connecting after" | tee output.log
         if grep -q 'insert_a_suppression_name_here' output.log; then echo "Valgrind found problems"; exit 1; fi
 
+    - name: Run Test with aux lib
```

Comment:
don't need additionally steps for force aux funcs, instead, append them to existing tests (Line 196 and Line 210)

Reply:
updated